src/components/connections/Connection.ts

```diff
@@ -17,16 +21,14 @@ export interface InfluxDBConnection {
   isActive: boolean;
 }
 
-export function emptyInfluxDBConnection(): InfluxDBConnection {
-  return {
-    version: InfluxConnectionVersion.V2,
-    id: "",
-    name: "",
-    hostNport: "",
-    token: "",
-    org: "",
-    isActive: false
-  };
+export const emptyInfluxDBConnection: InfluxDBConnection = {
+  version: InfluxConnectionVersion.V2,
+  id: "",
+  name: "",
+  hostNport: "",
```

Comment:
should be `host` here

Reply:
we reverted back to hostNport for the time being

Reply:
Cool